cookbook/composer.rst

```diff
@@ -14,35 +14,24 @@ following sections.
 Install Composer on Linux and Mac OS X
 --------------------------------------
 
-To install Composer on Linux or Mac OS X, execute the following two commands:
-
-.. code-block:: bash
-
-    $ curl -sS https://getcomposer.org/installer | php
-    $ sudo mv composer.phar /usr/local/bin/composer
-
-.. note::
-
-    If you don't have ``curl`` installed, you can also just download the
-    ``installer`` file manually at https://getcomposer.org/installer and
-    then run:
+#. Run the installer as described in `the official Composer documentation`_;
+#. Execute the following command to move the ``composer.phar`` to a directory that is in your path:
 
     .. code-block:: bash
-
-        $ php installer
+    
         $ sudo mv composer.phar /usr/local/bin/composer
```

Comment:
The whole code block must be indented by one space less to not be rendered as a quote.